src/ArchiveFolder.cs

```diff
@@ -74,25 +83,19 @@ public async Task<IChildFile> CreateFileAsync(string name, bool overwrite = fals
 
     protected override ReadOnlyArchiveFolder WrapSubfolder(string name) => new ArchiveFolder(this, name);
 
-    protected async Task RemoveSubfolder(IArchiveEntry entry, CancellationToken cancellationToken = default)
```

Comment:
This method wasn't used anywhere, so it didn't seem worth updating it to fix this bug.